lib/extensions.js

```diff
@@ -81,16 +81,43 @@ function getBinaryName() {
  * is set, return that path. Otherwise make
  * path using current release version and
  * binary name.
+ * 
+ * Optionally, the default URL can be replaced by
+ * setting the environment variable SASS_BINARY_SITE
+ * (or commandline option --sass-binary-site)
+ * which should point to the mirror of the repository
+ * laid out as follows:
+ *
+ * SASS_BINARY_SITE/
+ * 
+ *  v3.0.0-beta.4
+ *  v3.0.0-beta.4/freebsd-x64-14_binding.node
+ *  v3.0.0-beta.5
+ *  v3.0.0-beta.5/freebsd-ia32-11_binding.node
+ *  v3.0.0-beta.5/freebsd-x64-42_binding.node
+ *  v3.0.0-beta.5/freebsd-x64-14_binding.node
+ *  v3.0.0-beta.5/freebsd-x64-11_binding.node
+ *  v3.0.0-beta.5/freebsd-ia32-43_binding.node
+ *  v3.0.0-beta.5/freebsd-ia32-42_binding.node
+ *  v3.0.0-beta.5/freebsd-ia32-14_binding.node
+ *  v3.0.0-beta.5/freebsd-x64-43_binding.node
+ *
+ * ... etc. for all supported versions and platforms
  *
  * @api private
  */
 
 function getBinaryUrl() {
   return flags['--sass-binary-url'] ||
-         package.nodeSassConfig ? package.nodeSassConfig.binaryUrl : null ||
+         (package.nodeSassConfig ? package.nodeSassConfig.binaryUrl : null) ||
```

Comment:
"Braces" are usually `{}`, I think you mean `()`?

Check this out (on current `master`):

````
m> node scripts/install.js --sass-binary-url http://www.google.com/
/home/saper/sw/test/node_modules/node-sass/lib/extensions.js:90
         package.nodeSassConfig ? package.nodeSassConfig.binaryUrl : null ||
                                                        ^
TypeError: Cannot read property 'binaryUrl' of undefined
    at getBinaryUrl (/home/saper/sw/test/node_modules/node-sass/lib/extensions.js:90:57)
    at Object.<anonymous> (/home/saper/sw/test/node_modules/node-sass/lib/extensions.js:114:18)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/home/saper/sw/test/node_modules/node-sass/scripts/install.js:11:1)
    at Module._compile (module.js:460:26)
````